src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,53 @@
+.catalog {
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
+
+@media (max-width:488px) {
+  .catalog {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(1, 200px);
+    gap: 48px;
+  }
+}
+
+@media (min-width:489px) and (max-width:768px) {
+  .catalog {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(2, 200px);
+    gap: 48px;
+  }
+}
+
+@media (min-width:769px) and (max-width:1024px) {
+  .catalog {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(3, 200px);
+    gap: 48px;
+  }
+}
+
+@media (min-width:1024px) {
+  .catalog {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(4, 200px);
+    gap: 48px;
+  }
+}
```

Comment:
When making changes to the rules in the media, it is important to only include the ones that require modification. Avoid duplicating styles and ensure that if you are only altering the 'grid-template-columns' rule, it is the sole one present in the media.